design/one-pager-stacks-versioning.md

```diff
@@ -0,0 +1,248 @@
+# Stack Versioning
+* Owner: Luke Weber (@lukeweber)
+* Reviewers: Crossplane Maintainers
+* Status: Draft, revision 1.0
+
+## Problem
+
+Stacks installer currently relies on installing a particular version of stack from a docker tag. The stack installer in
```

Comment:
> a particular version of a stack

Is this referring to the version of the stack which is being installed, or is this referring to the version of the Crossplane Stack Schema which the stack adheres to?